label/bpe/train.py

```diff
@@ -0,0 +1,160 @@
+__all__ = ["TrainBPEModelJob", "ReturnnTrainBpeJob"]
+
+import gzip
+import subprocess as sp
+import os
+import sys
+
+from sisyphus import *
+
+import i6_core.util as util
+
+Path = setup_path(__package__)
+
+
+class TrainBPEModelJob(Job):
+    """
+    Create a bpe codes file using the official subword-nmt repo, either installed from pip
+    or https://github.com/rsennrich/subword-nmt
```

Comment:
Btw, you can do the same with the sentencepiece software. I'm not sure if there are any differences. If there are, I expect the sentencepiece software to work better. At least, it also should be faster.

Reply:
Yes sure, but I would not trust that this results in the exact same BPE codes that are compatible with Kazukis LM models.

Of course we could add a comment that you should use the BPE mode of the Sentencepiece software for new setups.

Reply:
Two ideas what we could (maybe should) do:

- At least mention this in the docstring here.
- Maybe rename the job to reflect that this is the job specifically for SubwordNMT.

The same applies for most other jobs here and in apply.py. I'm not repeating that now for all cases.

Reply:
Okay updated the docstrings and renamed the whole package to "subword_nmt" so that it is clear it is not about bpe but about this implementation in specific.